glib/src/collections/list.rs

```diff
@@ -658,7 +658,7 @@ pub struct IterMut<'a, T: TransparentPtrType> {
 
 impl<'a, T: TransparentPtrType> IterMut<'a, T> {
     #[inline]
-    fn new(list: &'a mut List<T>) -> IterMut<'a, T> {
```

Comment:
This is wrong. The iterator gives mutable access to the list so we need a mutable reference

Reply:
Apparently clippy also does not warn about this anymore.